demo/lib/App.tsx

```diff
@@ -96,6 +103,59 @@ export default class App extends React.Component<any, AppState> {
   };
 
   render() {
+
+    let customChildren = null;
+    if (this.state.customChildren) {
+      customChildren = ({ circularProps, linearProps, ...props }) => {
+        if (this.state.viewer === "linear") {
+          return  (
+              <div ref={this.linearRef} style={{ height: "70%", width: "100%" }}>
```

Comment:
```suggestion
              <div ref={this.linearRef} style={{ height: "100%", width: "100%" }}>
```

Can/should this be 100%?

Reply:
I was just testing to make sure 70% works.  but 100% definitely looks better!